### PR TITLE
Versioned clientlibs transformer factory

### LINKED ISSUE #866: Sling context path does not get stripped with versioned client libs
When a context path is set for AEM/Sling the versioned clientlib rewriter throws an error because the resource path is invalid

`com.adobe.granite.ui.clientlibs.impl.HtmlLibraryManagerImpl No library configured at /context-path/etc/clientlibs/mylib`

### PR DESCRIPTION
**Pull request by royteeuwen**
Fixed a bug for the versioned clientlibs transformer factory when a context path is set

Resolves #866

### CODE REVIEW

**Comment by bdhoine** [comment]
Thanks, we really needed this 👍

File: bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java [hunk 1]
```diff
@@ -129,25 +131,33 @@ public Transformer createTransformer() {
         return new VersionableClientlibsTransformer();
     }
 
-    private Attributes versionClientLibs(final String elementName, final Attributes attrs) {
+    private Attributes versionClientLibs(final String elementName, final Attributes attrs, String contextPath) {
```
Comment (bdhoine) [inline bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java]: make String final

Comment (royteeuwen) [inline bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java]: done

Comment (wiiitek) [inline bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java]: There are two methods with `String contextPath` argument but it is made `final` in only one of them.
Is it ok?

**Comment by royteeuwen** [comment]
Fixed :)

**Comment by justinedelson** [comment]
Thanks!

**Pull request merged**